### PR TITLE
Fix the issue with Nvidia pubkey

### PR DESCRIPTION
**Pull request by renn0xtek9**
This shall fix the issue made by Nvidia changing its public key 
see this https://forums.developer.nvidia.com/t/notice-cuda-linux-repository-key-rotation/212772

### CODE REVIEW

File: Dockerfile.melodic [hunk 1]
```diff
@@ -4,67 +4,81 @@ ARG CARLA_VERSION=0.9.8
 ENV NVIDIA_VISIBLE_DEVICES ${NVIDIA_VISIBLE_DEVICES:-all}
 ENV NVIDIA_DRIVER_CAPABILITIES ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
 
+#Fix the issue with broken Nvidia plublich key
```
Comment (renn0xtek9) [inline Dockerfile.melodic:7]: This section fixes the issue with the pub key

**Comment by atinfinity** [comment]
@renn0xtek9 Thank you for your contribution! I'll review and check. Please wait a little.

**Comment by atinfinity** [comment]
@renn0xtek9 I posted a question to NVIDIA developer in forum. Please wait these answer.
https://forums.developer.nvidia.com/t/updating-the-cuda-linux-gpg-repository-key/212897/22

File: Dockerfile.melodic (hunk 1 above)
Comment (atinfinity) [reply Dockerfile.melodic:7]: OK

**Comment by atinfinity** [comment]
@renn0xtek9 

> I posted a question to NVIDIA developer in forum. Please wait these answer.
> <https://forums.developer.nvidia.com/t/updating-the-cuda-linux-gpg-repository-key/212897/22>

NVIDIA's answer is here <https://forums.developer.nvidia.com/t/updating-the-cuda-linux-gpg-repository-key/212897/23>.
The Dockerfile of this project used CUDA 10.2 Docker image as base image. So, I need to fix it.

And, I think that this change is better. Because layer od docker is smaller than current change..  
<https://forums.developer.nvidia.com/t/updating-the-cuda-linux-gpg-repository-key/212897/30>

**Comment by renn0xtek9** [comment]
Yes, sounds better indeed.

**Comment by atinfinity** [comment]
@renn0xtek9 I resolved in <https://github.com/atinfinity/carla_ros_bridge_docker/pull/4>. So, I closed this PR.

**Pull request closed without merging**